From 090140752d4cad1cb7bbae2c3de12fc29115b2d6 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:04:29 -0400
Subject: [usb] ehci: Remove debugging at every interrupt

Message-id: <1410887463-20674-198-git-send-email-dzickus@redhat.com>
Patchwork-id: 95554
O-Subject: [RHEL7 PATCH 197/591] ehci: Remove debugging at every interrupt
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 991fb3daac946441dbf4c237b2dd4d6c11fff633
Author: Oliver Neukum <oneukum@suse.de>
Date:   Mon Nov 18 13:23:02 2013 +0100

    ehci: Remove debugging at every interrupt

    This is overkill. Just removeit.

    Signed-off-by: Oliver Neukum <oneukum@suse.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/host/ehci-hcd.c b/drivers/usb/host/ehci-hcd.c
index e3afac7..d0a31a2 100644
--- a/drivers/usb/host/ehci-hcd.c
+++ b/drivers/usb/host/ehci-hcd.c
@@ -714,13 +714,6 @@ static irqreturn_t ehci_irq (struct usb_hcd *hcd)
  cmd = ehci_readl(ehci, &ehci->regs->command);
  bh = 0;
 
-#ifdef VERBOSE_DEBUG
- /* unrequested/ignored: Frame List Rollover */
- dbg_status (ehci, "irq", status);
-#endif
-
- /* INT, ERR, and IAA interrupt rates can be throttled */
-
  /* normal [4.15.1.2] or error [4.15.1.1] completion */
  if (likely ((status & (STS_INT|STS_ERR)) != 0)) {
   if (likely ((status & STS_ERR) == 0))
-- 
1.7.1